DOCKER.md

```diff
@@ -0,0 +1,44 @@
+# Running SDGym using docker
+We support using Docker to run SDGym. This can help you avoid dependency issues when running SDGym
+on Windows as well as in environments where you don't have permission to install libraries.
+
+## Basic Usage
+To get started with SDGym on Docker, you should do the following:
+
+1. Pull the [latest SDGym image](https://hub.docker.com/r/sdvproject/sdgym) from DockerHub by running:
+
+```bash
+docker pull sdvproject/sdgym <tag>
```

Comment:
Shouldn't this be `docker pull sdvproject/sdgym:<tag>`  (so, `:` between the image name and the tag)